ele/lib/elements.json

```
@@ -3,708 +3,944 @@
   "terbium": {
     "name": "terbium",
     "symbol": "Tb",
     "atomic number": 65,
-    "mass": 158.93
+    "mass": 158.93,
+    "radius_bondi": null,
+    "radius_alvarez": 2.79
   },
   "dysprosium": {
     "name": "dysprosium",
     "symbol": "Dy",
     "atomic number": 66,
-    "mass": 162.50
+    "mass": 162.50,
+    "radius_bondi": null,
+    "radius_alvarez": 2.87
   },
   "holmium": {
     "name": "holmium",
     "symbol": "Ho",
     "atomic number": 67,
-    "mass": 164.93
+    "mass": 164.93,
+    "radius_bondi": null,
+    "radius_alvarez": 2.81
   },
   "erbium": {
     "name": "erbium",
     "symbol": "Er",
     "atomic number": 68,
-    "mass": 167.26
+    "mass": 167.26,
+    "radius_bondi": null,
+    "radius_alvarez": 2.83
   },
   "thulium": {
     "name": "thulium",
     "symbol": "Tm",
     "atomic number": 69,
-    "mass": 168.93
+    "mass": 168.93,
+    "radius_bondi": null,
+    "radius_alvarez": 2.79
   },
   "ytterbium": {
     "name": "ytterbium",
     "symbol": "Yb",
     "atomic number": 70,
-    "mass": 173.05
+    "mass": 173.05,
+    "radius_bondi": null,
+    "radius_alvarez": 2.80
   },
   "lutetium": {
     "name": "lutetium",
     "symbol": "Lu",
     "atomic number": 71,
-    "mass": 174.97
+    "mass": 174.97,
+    "radius_bondi": null,
+    "radius_alvarez": 2.74
   },
   "hafnium": {
     "name": "hafnium",
     "symbol": "Hf",
     "atomic number": 72,
-    "mass": 178.49
+    "mass": 178.49,
+    "radius_bondi": null,
+    "radius_alvarez": 2.63
   },
   "tantalum": {
     "name": "tantalum",
     "symbol": "Ta",
     "atomic number": 73,
-    "mass": 180.95
+    "mass": 180.95,
+    "radius_bondi": null,
+    "radius_alvarez": 2.53
   },
   "tungsten": {
     "name": "tungsten",
     "symbol": "W",
     "atomic number": 74,
-    "mass": 183.84
+    "mass": 183.84,
+    "radius_bondi": null,
+    "radius_alvarez": 2.57
   },
   "rhenium": {
     "name": "rhenium",
     "symbol": "Re",
     "atomic number": 75,
-    "mass": 186.21
+    "mass": 186.21,
+    "radius_bondi": null,
+    "radius_alvarez": 2.49
   },
   "osmium": {
     "name": "osmium",
     "symbol": "Os",
     "atomic number": 76,
-    "mass": 190.23
+    "mass": 190.23,
+    "radius_bondi": null,
+    "radius_alvarez": 2.48
   },
   "iridium": {
     "name": "iridium",
     "symbol": "Ir",
     "atomic number": 77,
-    "mass": 192.22
+    "mass": 192.22,
+    "radius_bondi": null,
+    "radius_alvarez": 2.41
   },
   "platinum": {
     "name": "platinum",
     "symbol": "Pt",
     "atomic number": 78,
-    "mass": 195.08
+    "mass": 195.08,
+    "radius_bondi": 1.72,
+    "radius_alvarez": 2.29
   },
   "gold": {
     "name": "gold",
     "symbol": "Au",
     "atomic number": 79,
-    "mass": 196.97
+    "mass": 196.97,
+    "radius_bondi": 1.66,
+    "radius_alvarez": 2.32
   },
   "mercury": {
     "name": "mercury",
     "symbol": "Hg",
     "atomic number": 80,
-    "mass": 200.59
+    "mass": 200.59,
+    "radius_bondi": 1.70,
```

bondi table has 1.55
```suggestion
    "radius_bondi": 1.55,
```

---

I think Zn, Cd, Hg, and Pb come from the 1966 paper (Table 1)

---

Yes! Check the notes describing the sources to make sure this is all stated clearly :)